shared/actions/fs/index.tsx

```diff
@@ -898,7 +934,7 @@ function* fsSaga() {
   )
   if (flags.kbfsOfflineMode) {
     yield* Saga.chainAction2(FsGen.setTlfSyncConfig, setTlfSyncConfig)
-    yield* Saga.chainAction2([FsGen.loadTlfSyncConfig, FsGen.loadPathMetadata], loadTlfSyncConfig)
```

Comment:
why delete this?

Reply:
It's already part of the Tlf, either from `favoritesLoad` or `loadAdditionalTlf`. So no need to load it separately.